src/theory/quantifiers/inst_match_trie.h

```diff
@@ -131,7 +132,8 @@ class CDInstMatchTrie
    * equalities in the equality engine of qs.
    * It additionally takes a context c, for which the entry is valid in.
    */
-  bool existsInstMatch(QuantifiersState& qs,
+  bool existsInstMatch(Env& env,
```

Comment:
Can you pass user context instead of Env here?  I'm actually planning to simplify this method so that is furthermore does not depend on `qs` either.

Reply:
Makes sense, done!